doc/source/user_guide/session/launching_ansys_fluent.rst

```diff
@@ -300,6 +300,50 @@ can be launched explicitly using :func:`create_launcher() <ansys.fluent.core.lau
   >>> pim_solver_session = pim_solver_launcher()
 
 
+Launching Fluent in Container Mode with Docker Compose or Podman Compose
+------------------------------------------------------------------------
+
+Use PyFluent with Docker Compose or Podman Compose to run Fluent in a consistent, reproducible containerized environment.
+
+1. **Docker Compose**
```

Comment:
@mkundu1 The use of numbers here implies a sequential order. Is that true or should bullets be used?

Reply:
It's a sequential order.